src/core/Makefile.am

```diff
@@ -89,6 +87,13 @@ SOURCES_COMMON                      = \
     utils/slaac_address.cpp           \
     $(NULL)
 
+if OPENTHREAD_ENABLE_MAC_WHITELIST
```

Comment:
I believe these files should already be included in the windows build and the configure option is set to `1`.  What am I missing?